src/string_immutable.zig

```diff
@@ -3233,11 +3233,13 @@ pub fn copyUTF16IntoUTF8WithBuffer(buf: []u8, comptime Type: type, utf16: Type,
             if (bun.FeatureFlags.use_simdutf) {
                 log("UTF16 {d} -> UTF8 {d}", .{ utf16.len, out_len });
                 if (remaining.len >= out_len) {
-                    const result = bun.simdutf.convert.utf16.to.utf8.with_errors.le(trimmed, remaining);
+                    // using trimmed on windows can give us STATUS_STACK_BUFFER_OVERRUN
+                    const read = if (Environment.isWindows) utf16 else trimmed;
```

Comment:
This is not a fix. Why is trimmed causing stack buffer overrun?

Reply:
Will remove from this PR, need to investigate `bun.simdutf.trim.utf16` looks like is returning some broke utf16 with makes simdutf8 panic with `STATUS_STACK_BUFFER_OVERRUN` on windows (probably not windows only bug)

Reply:
reverted will be solved on another PR because looks like a path converttion bug